.githooks/pre-commit

```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env bash
 
-cargo make check-rs-format && \
-cargo make check-ts-format
+set -e                          # grep failure does not trigger exit.
+
+debug=0                         # Set to 1 to enable debugging, 0 to disable.
```

Comment:
```suggestion
dry=0
```
It's more of a _dry_ option than a _debug_ one.

Reply:
Implemented as suggested.